baklava/README.md

```diff
@@ -15,24 +15,26 @@ It uses ElectronJS as a wrapper for [dogehouse.tv](https://dogehouse.tv) and add
 On the latest additions that allow the use of global keybinds a few things have to be noted:
 
 - Specify your Nodejs ABI number in `package.json` under `iohook/targets`
-   - Run `node -v` and note down your nodejs version
-   - Note down the `NODE_MODULE_VERSION` from [here](https://nodejs.org/en/download/releases/) by referring your previouly noted down nodejs version
-   - In `package.json` add `node-%NODE_MODULE_VERSION%` to `iohook/targets`
+
+  - Run `node -v` and note down your nodejs version
+  - Note down the `NODE_MODULE_VERSION` from [here](https://nodejs.org/en/download/releases/) by referring your previouly noted down nodejs version
+  - In `package.json` add `node-%NODE_MODULE_VERSION%` to `iohook/targets`
 
 - Specify your electron ABI in `package.json` under `iohook/targets`
-   - Run an electron app using `npm run start` without importing `iohook` *(i.e. remove/comment all imports of `iohook` from `src/electron.ts` and `src/util.ts`)*
-   - Open the dev console and type `process.versions.modules`
-   - In `package.json` add `electron-%process.versions.modules%` to `iohook/targets`
-   - Now uncomment/add the previously removed `iohook` imports
+
+  - Run an electron app using `npm run start` without importing `iohook` _(i.e. remove/comment all imports of `iohook` from `src/electron.ts` and `src/util.ts`)_
+  - Open the dev console and type `process.versions.modules`
+  - In `package.json` add `electron-%process.versions.modules%` to `iohook/targets`
+  - Now uncomment/add the previously removed `iohook` imports
 
 - Run `npm i iohook`
-- Run `npm run start` and you global keybinds *(including PTT)* should start working
-- *(Optional)* Run `npm run build:%YOUR_PLATFORM_CODE%` and install the app from the build *(located in `/builds`)*
+- Run `npm run start` and you global keybinds _(including PTT)_ should start working
+- _(Optional)_ Run `npm run build:%YOUR_PLATFORM_CODE%` and install the app from the build _(located in `/builds`)_
```

Comment:
Don't think you need to replace the `*` with `_`

Reply:
It happened because of the prettier formatter I use in vscode. I will fix it no problem.

Reply:
Fixed it

Reply:
Oh lol i didn't mean to order you to "Fix it now" i was just asking why doing these small pointless changes. But it's fine even if you reverted them back!